src/tribol/mesh/MfemData.hpp

```diff
@@ -1168,6 +1192,12 @@ class MfemMeshData {
    */
   ParentField coords_;
 
+  /**
+   * @brief Contains reference coords grid function and transfer operators if
```

Comment:
Likely due to my lack of familiarity, but the doxygen doesn't make it clear how to access or use `reference_coords_` to get the coordinates (ordering?) or how this pointer may contain transfer operators. Can you clarify?

Reply:
It's a class that wraps the fields on parent, submesh, LOR, and redecomp meshes and facilitates transfer between them.  See the def for the `ParentField` class for more info.